dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/ProviderConsumerRegTable.java

```diff
@@ -92,4 +92,29 @@ public static void registerConsumer(Invoker invoker, URL registryUrl, URL consum
         return invokers;
     }
 
+    public static boolean isRegistered(String serviceUniqueName) {
+        Set<ProviderInvokerWrapper> providerInvokerWrapperSet = ProviderConsumerRegTable.getProviderInvoker(serviceUniqueName);
+        for (ProviderInvokerWrapper providerInvokerWrapper : providerInvokerWrapperSet) {
```

Comment:
can be rewritten as 
return providerInvokerWrapperSet.stream().anyMatch(ProviderInvokerWrapper::isReg);

Reply:
done.